api/envoy/extensions/transport_sockets/tls/v3/tls.proto

```diff
@@ -299,4 +299,10 @@ message CommonTlsContext {
   // Custom TLS handshaker. If empty, defaults to native TLS handshaking
   // behavior.
   config.core.v3.TypedExtensionConfig custom_handshaker = 13;
+
+  string tls_key_log_path = 15;
+
+  string tls_key_log_src = 16;
```

Comment:
Why don't we use `repeatedly` here?

Reply:
The three parameters are different fields. How can we use repeated ?